### ISSUE TITLE
publish this software on f-droid, please

### ISSUE THREAD

**Issue by githare** [duniter/duniter-android-app#14]
Since I do not want to use the play store and I'd like to update automatically to new versions, would it be possible to publish this software on f-droid, please?

**Comment by blavenie** [comment]
FDroid send this response :

If you want to go further, you can provide merge requests against gitlab.com/fdroid/fdroidata . I haven't looked at the code in details, because it uses Google's play-services as dependency, which is non-free and cannot be built by us.

Post Link: https://f-droid.org/forums/topic/ucoin-apps/#post-16740

**Comment by sbernard31** [comment]
I do not use the play store too. I think F-Droid is a good place for ucoin-android-app.
I don't know why you need Google's play-services ? But I think this should be a good thing to remove all "non free" google apps dependency. If we want a money in which we trust, we also need software in which we trust. And I really don't trust in proprietary Google code.

Maybe this could help : https://f-droid.org/wiki/page/FAQ_-_App_Developers

**Comment by blavenie** [comment]
Yes, we need to remove this dependency : com.google.android.gms.play-services.

@mmpio : Why did you add this lib ?

**Comment by mmpio** [comment]
This lib has been there since the beginning, 
( commit c0ab312f1601cebb41cca0d216f945afe6470d44 ) 
and I'm not the one who put it there so I don't know why it is.

**Comment by blavenie** [comment]
The unexpected lib has been removed. So i will try a new post on FDroid

**Comment by blavenie** [comment]
FDroid contribe manual : https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md

**Comment by jpggithub** [comment]
I'm looking for news about duniter app in f-droid repository to test testnet but I'm finding nothing on f-droid.org.
Is it a work in progress?
